server/config.py

```diff
@@ -23,69 +65,11 @@ class EnvironmentConfig:
         'access_token_url': 'https://www.openstreetmap.org/oauth/access_token',
         'authorize_url': 'https://www.openstreetmap.org/oauth/authorize'
     }
+
+    # Some more definitions (not overridable)
+    API_DOCS_URL = f'{APP_BASE_URL}/api-docs/swagger-ui/index.html?' + \
+                   f'url={APP_BASE_URL}/api/docs'
     SEND_FILE_MAX_AGE_DEFAULT = 0
-    SQLALCHEMY_DATABASE_URI = os.getenv('TM_DB', None)
```

Comment:
should we consider checking for this variable as an alternative to retain backwards compatibility?

Reply:
i guess we have a few other changes on here (e.g. removing the separate environments) so perhaps we just say this will be a point where there are breaking changes and your configuration must be updated in a few ways.

Reply:
Yes, the idea is to have one place where to configure the TM. I hope this will make things easier for people to get it set up.

Reply:
Let's point out a bit more the "breaking" parts.

As far as I see there are two essential things breaking something:

* `TM_DB` - Which is replaced by the `POSTGRES_X` variables to unify with the database configuration (docker container). If wished, we could maintain backwards compatibility. I'd be happy to code the couple of extra lines to handle it.
* `TM_ENV` - Which is becoming deprecated with this PR. It was pushing our very custom configuration for `https://tasks.hotosm.org` and `https://tasks-stage.hotosm.org`. Deprecating this should not effect anybody apart from our main TM instance. Here are `.env` files for both setups that we can use: [tasks.hotosm.org](https://gist.github.com/xamanu/5f30378e4e3294d465463b209a49fce3) and [tasks-stage.hotosm.org](https://gist.github.com/xamanu/9060531ae91109076f4e20b769d88a11).

Is there anything, that I'm missing? @ethan-nelson, @dakotabenjamin and @arunasank, do you prefer to be conservative on the `TM_DB1` variable and maintain backwards compatibility?  I really don't want to make this an annoying change. Just fixing things to get the Docker stuff right, to ease up installation and allow non-devs to test PRs.